config/buildspec_vanilla_framework_tests.yml

```diff
@@ -20,7 +20,7 @@ phases:
         - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
         - cd $CODEBUILD_SRC_DIR  && chmod +x config/protoc_downloader.sh && ./config/protoc_downloader.sh
         - pip install --upgrade pip==20.3.3
-        - pip install -q -U pytest pytest-cov wheel pyYaml pytest-html keras==2.3.1 mxnet torch xgboost pre-commit tensorflow_datasets==4.0.1 torchvision
+        - pip install -q -U pytest pytest-cov wheel pyYaml pytest-html keras==2.3.1 mxnet==1.6.0 torch xgboost pre-commit tensorflow_datasets==4.0.1 torchvision
```

Comment:
Let's pin the versions to the other frameworks too.

Reply:
Done